src/main/java/uk/ac/cam/cl/dtg/segue/comm/EmailManager.java

```diff
@@ -154,7 +155,7 @@ public void sendTemplatedEmailToUser(final RegisteredUserDTO userDTO, final Emai
 
         // default properties
         //TODO: We should find and replace this in templates as the case is wrong.
```

Comment:
Can we arrange to do this prior to this merge and therefore remove all reference to givenname